packages/web3torrent/package.json

```diff
@@ -23,6 +23,7 @@
     "webtorrent": "0.107.16"
   },
   "devDependencies": {
+    "@statechannels/channel-provider": "*",
```

Comment:
Should we have this as a hard dependency for now, even if that isn't the long term vision?

Reply:
Yes I think so